src/components/Details.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useLocation, NavLink } from 'react-router-dom';
+import { BsArrowLeftCircle } from 'react-icons/bs';
+
+const Details = () => {
+  const location = useLocation();
+  const { state } = location;
+  return (
+    <section className="container details-card">
+      <NavLink
+        className="align-self-center arrow"
+        to="/"
```

Comment:
- [ ] From the design this arrow does not belong here. It should be in the Header section. Kindly stick to the design as much as possible.

  Your arrow | Design arrow
  :---:|:---:
  ![arr-1](https://user-images.githubusercontent.com/36542986/189102173-0e6b6868-c2cc-4b95-9eaa-8109eb613083.jpg) | ![arr](https://user-images.githubusercontent.com/36542986/189102177-595e6f9a-587e-4b13-8b1f-5262fbbe40f1.jpg)